docs/get-started.md

```diff
@@ -20,7 +20,7 @@ For more details on how to create repositories using template, read the article
 
 ## After creating your repository
 
-The following steps apply to your new repository, not the upptime/upptime template repository.
+The following steps apply to your new repository, not the `upptime/upptime` template repository.
 
 ### Enable publishing
```

Comment:
I've updated this to reflect the changes made to the GitHub Pages Settings page.